boms/modeshape-bom-embedded/pom.xml

```diff
@@ -366,7 +366,7 @@
             </dependency>
             <!--Required by infinispan-cachestore-jdbc -->
             <dependency>
-                <groupId>c3p0</groupId>
+                <groupId>com.mchange</groupId>
```

Comment:
I'm curious, did this c3p0 dependency change in Infinispan between CR1 and Final?